nengo_viz/static/viz_netgraph.js

```diff
@@ -104,6 +131,7 @@ VIZ.NetGraph = function(parent, args) {
             self.offsetY = (self.offsetY + yy) / scale - yy;
             
             self.scale = scale * self.scale;
+            interact.margin(30*self.scale);
```

Comment:
I think this is why @celiasmith wanted to make the objects movable by dragging the label instead? When an object is super small, it's kind of impossible to allow two separate behaviors, moving and resizing. If you allow moving, then you can never resize. At least by making resizing the only possibility, you can resize, move, then resize again.